### ISSUE TITLE
it work with reactNative? 

### ISSUE THREAD

**Issue by tarangshah19** [BALKANGraph/FamilyTreeJS#95]
it work with reactNative?

**Comment by ZornitsaPesheva** [comment]
Yes, it work in any environment that can run JavaScript, but we don't have examples for Reat Native.

**Comment by tarangshah19** [comment]
not working 
![image](https://github.com/BALKANGraph/FamilyTreeJS/assets/73467634/ca73e971-c515-4f43-9154-e34ac4ba95cd)

**Comment by ZornitsaPesheva** [comment]
Can you at all use document.querySelector in Ract Native?

**Comment by tarangshah19** [comment]
in react native Dom is not available we cant use documents

**Comment by ZornitsaPesheva** [comment]
You need WebView to work with js in the browser:
https://github.com/react-native-webview/react-native-webview/blob/master/docs/Guide.md

**Comment by tarangshah19** [comment]
need to use js file or react code ?

**Comment by ZornitsaPesheva** [comment]
You will need to use orgchart.js in your React code.

**Comment by ZornitsaPesheva** [comment]
Yes, Family Tree JS is frontend library. You need WebViewto display web content in React Native app

**Comment by tarangshah19** [comment]
i dont thing paid version help full then in in react native if work with webview only

**Comment by ZornitsaPesheva** [comment]
Most probably you need WebView for every web based JS library.

**Comment by tarangshah19** [comment]
but using d3 js i can make this without WebView but required lot of time so i thinking about your plugin

**Comment by ZornitsaPesheva** [comment]
Yes, they have React component.
We don't use frameworks as React and haven't developed such components for now.

**Comment by tarangshah19** [comment]
ok thanks for guid me

**Comment by tarangshah19** [comment]
if i get paid version can you provide me react native components?

**Comment by ZornitsaPesheva** [comment]
Can you give us a link to that d3 library that you can use in React Native without WebView?

**Comment by tarangshah19** [comment]
![WhatsApp Image 2024-01-06 at 11 56 07 AM](https://github.com/BALKANGraph/FamilyTreeJS/assets/73467634/6d44d2e4-199a-4ceb-b4d7-b68160fa49d0)
d3 hierarchy we are using

**Comment by ZornitsaPesheva** [comment]
As I understand in your current application you don't use React Native.
But in one of the previous comment you are telling us that you can create a React Native app using d3 without WebView.
I am asking you how you are gonna do this?
We need an example for D3 in React Native without WebView.

**Comment by tarangshah19** [comment]
hi sorry for replying late we do not have any examples now just want to know if we buy can you give us react-native components without web-view (Just want without DOM manipulation)

**Comment by ZornitsaPesheva** [comment]
We don't know if this is possible at all. If you show us how this is done with some similar library, we could check.

**Comment by tarangshah19** [comment]
ok i will update you on this

**Comment by tarangshah19** [comment]
hi,
hear is one example that i found in react native d3 

https://github.com/Johncy1997/react-native-family-tree

**Comment by tarangshah19** [comment]
https://github.com/Petrando/viscene-share
this repo work with react native and using d3 and without dom

**Comment by ZornitsaPesheva** [comment]
What do you mean by "works without dom"?
It runs in a browser and it is in a div element just as the Family Tree JS.
